### PR TITLE
Interface and additional class for repositories preparation was created

### PR DESCRIPTION
**Pull request by valentk777**

### CODE REVIEW

File: GREEDY/Models/Receipt.cs [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ﻿using System;
+using GREEDY.Interfaces;
```
Comment (MariusKurbakovas) [inline GREEDY/Models/Receipt.cs:2]: Importing an interface, doesn't make a class use it

File: GREEDY/Controllers/CreateAPathforDataController.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GREEDY.Controllers
+{
+    class CreateAPathforDataController
```
Comment (MariusKurbakovas) [inline GREEDY/Controllers/CreateAPathforDataController.cs:11]: In my opinion the proper name for this class would be CreatePathForDataController (The class name shouldn't use an article to make it shorter, and the word for should start with upper case)

**Review by MariusKurbakovas — COMMENTED** [review]
We should start creating new branches after each pull request, because using the old branches adds old commit messages

**Review by MariusKurbakovas — APPROVED** [review]
Commented on everything I thought was a little wrong, but all in all pretty good. You can merge this one and fix the changes in the next commit.

File: GREEDY/Controllers/CreateAPathforDataController.cs (hunk 2 above)
Comment (valentk777) [reply GREEDY/Controllers/CreateAPathforDataController.cs:11]: thanks. File was updated 👍

**Pull request merged**